neqo-transport/src/tracking.rs

```diff
@@ -500,7 +501,7 @@ impl RecvdPackets {
 
         tokens.push(RecoveryToken::Ack(AckToken {
             space: self.space,
-            ranges,
+            ranges: ranges.into_vec(),
```

Comment:
This could also be a SmallVec, couldn't it?

Reply:
Hmm, it seems like that is a good reason not to use `SmallVec` there.  Either that or do the boxing it suggests.  The net effect would be the same either way. `Vec` is heap allocation, `Box<SmallVec>` is a more complicated way of saying the same thing.  Maybe comment to that effect somewhere.

Reply:
The boxed slice seems like a good compromise.